pages/linux/flatpak-builder.md

```diff
@@ -0,0 +1,24 @@
+# flatpak-builder
+
+> Help build application dependencies.
+> More information: <https://docs.flatpak.org/en/latest/flatpak-builder-command-reference.html>.
+
+- Build a Flatpak then export it to a new repository:
```

Comment:
```suggestion
- Build a Flatpak and export it to a new repository:
```
Not sure about this, but I like this one a little better. (Same for example 2, 3 and 5)

Reply:
Ahh, yeah that makes sense.
I actually had "and" originally, but then replaced it for ", then".

I'll update it now.

Reply:
I think it can also be replaced in 4th example (line 18).